src/ext/CGAL/OGL_helper.h

```diff
@@ -25,7 +25,7 @@
 #include <CGAL/Simple_cartesian.h>
 #include <CGAL/Nef_3/SNC_decorator.h>
 
-#include "system-gl.h"
+#include "glview/system-gl.h"
```

Comment:
Note, this is the only place in src/ext that needed to be updated, but this is an explicitly [manually added OpenSCAD header](https://github.com/openscad/openscad/blame/f23b1525b9cc4ea3c7c654bff898392d9ef68b45/src/ext/CGAL/OGL_helper.h#L28) anyway.